run-system-tests/action.yml

```diff
@@ -98,7 +98,7 @@ runs:
         echo "HOME=/root" >> $GITHUB_ENV
         echo "CYPRESS_NETWORK_URL=$(echo ${{ inputs.network }})" >> $GITHUB_ENV
         
-        if [ "${{ inputs.run_smoke_tests }}" == "true" ];
+        if [ "${{ inputs.run_smoke_tests }}" != "false" ];
```

Comment:
probably this would be better comparision:
`if [ "$(echo "${{ inputs.run_smoke_tests }}" | tr '[:upper:]' '[:lower:]')" == "true" ];`